2023-11-17T00:24:03.2458241Z ##[group]Run actions/setup-python@v4
2023-11-17T00:24:03.2458802Z with:
2023-11-17T00:24:03.2459196Z   python-version: 3.8
2023-11-17T00:24:03.2459592Z   architecture: x64
2023-11-17T00:24:03.2459939Z   cache: pip
2023-11-17T00:24:03.2460333Z   check-latest: false
2023-11-17T00:24:03.2460867Z   token: ***
2023-11-17T00:24:03.2461380Z   update-environment: true
2023-11-17T00:24:03.2461860Z   allow-prereleases: false
2023-11-17T00:24:03.2462225Z ##[endgroup]
2023-11-17T00:24:03.3810744Z ##[group]Installed versions
2023-11-17T00:24:03.3887376Z Successfully set up CPython (3.8.18)
2023-11-17T00:24:03.3888953Z ##[endgroup]
2023-11-17T00:24:04.2525920Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-11-17T00:24:08.1216424Z /home/runner/.cache/pip
2023-11-17T00:24:09.8125275Z Received 6245959 of 6245959 (100.0%), 6.0 MBs/sec
2023-11-17T00:24:09.8126467Z Cache Size: ~6 MB (6245959 B)
2023-11-17T00:24:09.8153563Z [command]/usr/bin/tar -xf /home/runner/work/_temp/a589ce8a-c0cc-4e79-9ff0-2c8799e19ce3/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-11-17T00:24:09.8437136Z Cache restored successfully
2023-11-17T00:24:09.8458064Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
